DependencyInjection/Configuration.php

```diff
@@ -32,14 +32,12 @@ public function getConfigTreeBuilder()
             ->children()
                 ->arrayNode('directories')
                     ->info('List of directories relative to the kernel root directory containing classes.')
-                    ->useAttributeAsKey('prefix')
-                    ->prototype('array')
-                        ->prototype('scalar')->end()
-                    ->end()
+                    ->prototype('scalar')->end()
                     ->defaultValue([
-                        'controller' => ['../src/*Bundle/Controller', '../src/*Bundle/Action'],
-                        'command' => ['../src/*Bundle/Command'],
-                        'event_subscriber' => ['../src/*Bundle/EventSubscriber'],
+                        '../src/*Bundle/Controller',
+                        '../src/*Bundle/Action',
+                        '../src/*Bundle/Command',
+                        '../src/*Bundle/EventSubscriber',
```

Comment:
I'm not sure that we can automatically auto-discover all of these directories - it's very presumptuous and extra magical because you can't see it in config. This would certainly preclude it from being in the SE.